### ISSUE TITLE
ERROR when runing demo

### ISSUE THREAD

**Issue by niuwenju** [TNTWEN/OpenVINO-YOLOV4#1]
environment:
ubuntu18.04 64bit
openvino 2020.4

**Traceback (most recent call last):
  File "object_detection_demo_yolov3_async.py", line 33, in <module>
    import monitors
ModuleNotFoundError: No module named 'monitors'**

**Comment by TNTWEN** [comment]
Hello!@niuwenju 
When I first tried this demo, there was no ERROR. But when I tried the next day, I had the same problem. I tried many versions of monitors.
For example:
pip3 install monitors  
pip3 install python-monitors
But there are still many other problems with the use of this module.
So you could try   "pip3 install monitors" to install this module.  If there are any other errors about 'monitors' you could remove all code about monitors.
i will upload object_detection_demo_yolov3_async.py without using monitors in the assets folder. You could have a try!

**Comment by niuwenju** [comment]
thank you for sharing，i have remove all code about monitors then it runs well. there is an error as follows with monitors
**AttributeError: module 'monitors' has no attribute 'Presenter'**
version of monitors is v1.0.1

**Comment by TNTWEN** [comment]
Yeah. i have tried many versions of monitors. The error"AttributeError: module 'monitors' has no attribute 'Presenter'" still exists.i am still wondering which version of monitors the official demo use.And this demo's readme hasn't add details of these new functions

**Comment by niuwenju** [comment]
i have fixed it by using official monitors at path /opt/intel/openvino/deployment_tools/inference_engine/demos/python_demos/common

**Comment by TNTWEN** [comment]
@niuwenju
Thank you very much! it really works! i have added the monitors files  .You could verify these files and tell me if i made some mistakes.

**Comment by niuwenju** [comment]
now ,it works well.
